.readthedocs.yml

```diff
@@ -1,7 +1,9 @@
 version: 2
 
+formats: all
+
 python:
-    version: 3.7
-    install:
-      - requirements: requirements/docs.txt
-      - requirements: requirements/readthedocs.txt
+  version: 3.7
+  install:
+    - requirements: requirements/docs.txt
+    - requirements: requirements/readthedocs.txt
```

Comment:
why not merge the two?

Reply:
No idea... Other repos also have these two files. What is the difference between them?